CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ PrestaShop 1.6.x and 1.7.x
 ## [3.3.1] - 2020-12-03
 ### Added
  - Support for payment captures full reservation after discount is applied from the backend
-
+ - Code improvement
```

Comment:
The change log is for the end customer and should only contain things that are meaningful and add value to them.